<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicLrcPosterWidget</class>
 <widget class="QWidget" name="MusicLrcPosterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>歌词海报</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>600</width>
     <height>505</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>歌词海报</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>582</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>600</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#backgroundMask{background:rgba(255,255,255,228);}</string>
   </property>
   <widget class="QScrollArea" name="viewArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>275</width>
      <height>330</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MusicLrcPosterTableWidget" name="lrcArea">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>30</y>
      <width>275</width>
      <height>330</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="listArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>375</y>
      <width>470</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>393</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>打开图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>433</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>保存海报</string>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicLrcPosterTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musiclrcposterwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
